server/conn.go

```diff
@@ -849,7 +849,7 @@ func (cc *clientConn) writeResultset(goCtx goctx.Context, rs ResultSet, binary b
 }
 
 func (cc *clientConn) writeColumnInfo(columns []*ColumnInfo) error {
-	data := make([]byte, 4, 1024)
+	data := cc.alloc.AllocWithLen(4, 1024)
```

Comment:
We planned to remove arena allocator.

Reply:
Why would we remove arena allocator?

Reply:
Because we already use append for all the codec, a simple `[]byte` will do the same as arena allocator.

Reply:
The allocator interface is simple and clean. I prefer to use it for now.